### ISSUE TITLE
Service Provider Registration

### ISSUE THREAD

**Issue by markarsanious** [SEGUC17/Drop-The-Bit#22]
Severity: High
By: Mark Isaac
Description: We can’t sign up or apply as service providers (business) to join the system so we can’t register as businesses or test its functionalities
Steps: home
Expected: I should have the option to sign up as service provider or an option to apply as so and wait to be approved by a system admin.

**Comment by habiibaelghamry** [comment]
You will find written in the footer contact details where the business can send an email requesting to be added to the website. The web admin will then add the business and the business will receive an email with username and password.

**Comment by markarsanious** [comment]
I don't think that was the requested approach or at least nothing on the website direct businesses to do so or apply through this email.

**Comment by habiibaelghamry** [comment]
![image](https://cloud.githubusercontent.com/assets/25285151/25587999/d314fef6-2ea6-11e7-94a0-4b8dab97c0a0.png)
We thought this is clear enough.
